akka-docs/src/test/java/jdocs/cluster/singleton/ClusterSingletonSupervision.java

```diff
@@ -11,7 +11,6 @@
 
 // #singleton-supervisor-actor-usage-imports
 import akka.actor.PoisonPill;
-import akka.actor.Props;
```

Comment:
Another one in a code snippet in the docs, let's leave those in the test sources.